chain/cosmos/chain_node.go

```diff
@@ -831,7 +831,8 @@ func (tn *ChainNode) Exec(ctx context.Context, cmd []string, env []string) ([]by
 		Env:   env,
 		Binds: tn.Bind(),
 	}
-	return job.Run(ctx, cmd, opts)
+	err := job.Run(ctx, cmd, opts)
```

Comment:
Can you rename these value usages to `res` instead of `err`? There are a handful of them.

Reply:
Yeah, I'll get these changed.